src/tests/LocationDetails.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
This isnt related to this file, but your `ForecastDetails.test.js` is empty! You should NEVER include empty files in your repo. I presume this was just forgotten about before submission? It would be better if this was removed completely (and a note made in the README that adding further tests should be done in the future) than it be included in its present state